=== modified file 'src/session-service.c'
--- src/session-service.c	2010-03-18 14:22:41 +0000
+++ src/session-service.c	2010-03-26 15:35:18 +0000
@@ -637,7 +637,7 @@
 
 	if (can_suspend && allow_suspend) {
 		suspend_mi = dbusmenu_menuitem_new();
-		dbusmenu_menuitem_property_set(suspend_mi, DBUSMENU_MENUITEM_PROP_LABEL, _("Sleep"));
+		dbusmenu_menuitem_property_set(suspend_mi, DBUSMENU_MENUITEM_PROP_LABEL, _("Suspend"));
 		dbusmenu_menuitem_child_append(root, suspend_mi);
 		g_signal_connect(G_OBJECT(suspend_mi), DBUSMENU_MENUITEM_SIGNAL_ITEM_ACTIVATED, G_CALLBACK(machine_sleep), "Suspend");
 	}

